salvo/src/lib/benchmark/base_benchmark.py

```diff
@@ -148,9 +170,16 @@ class BenchmarkEnvironmentError(Exception):
 class BenchmarkEnvController():
   """Benchmark Environment Controller context class."""
 
+  _VARS_TO_CLEAR_AND_RESTORE = [
```

Comment:
Does this constant have to live on the class? Constants on classes have different lifetime which is sometimes desired, but most of the time it can confuse while debugging.

If there is nothing requiring us to package it with the class - let's just define it at the module level. We can do it just above the class to keep them close.

Reply:
Updated.